components/Header/index.tsx

```diff
@@ -32,6 +32,21 @@ const Header = () => {
   const mobileInputRef = useRef<HTMLInputElement>(null);
   const desktopInputRef = useRef<HTMLInputElement>(null);
 
+  const setShowSearch = (showSearch: boolean): void => {
```

Comment:
@ntorbinskiy you don't need to add TS types, when VS Code is smart enough to add it automatically:

![image](https://user-images.githubusercontent.com/46556479/194742410-4ae8d92f-7bac-4cc9-b276-522f4665bd06.png)